### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by LarsDenBakker** [webcomponents/webcomponentsjs#1028]
It would be useful to have a changelog for this library. I has bugs quite often, which is somewhat understandable given the complexity of the polyfill, so I have to clamp the version. But when clamping the version, It's hard to tell what I'm losing out.

**Comment by mttmccb** [comment]
Agree, just spotted a new version, had a look in the changelog and there's nothing there... 👎

**Comment by abdonrd** [comment]
Agree!
